public-api/src/main/java/org/pmiops/workbench/cdr/dao/AchillesAnalysisDao.java

```diff
@@ -9,7 +9,7 @@
     List<AchillesAnalysis> findAll();
 
     @Query(value = "select distinct a from AchillesAnalysis a left join FETCH a.results as r " +
-            "where r.stratum1 = ?1 and r.stratum2 in (?2) order by a.analysisId"
+            "where r.stratum1 = ?1 and r.stratum2 in (?2) and r.analysisId in (3110,3111,3112) order by a.analysisId"
```

Comment:
Is this supposed to get rid of questions that don't have results? Does it work? I don't think it will.  Because we are left joining to results. 

I was thinking the solution would be in the getSurveyQuestions dao call. Change that query to only return questions that have achilles results.

Reply:
Changed the query in questionconceptdao to fetch only the questions that have results.